tests/integration/components/point-test.js

```diff
@@ -0,0 +1,26 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+// module('Integration | Component | point', function(hooks) {
+//   setupRenderingTest(hooks);
+
+//   test('it renders', async function(assert) {
+//     // Set any properties with this.set('myProperty', 'value');
+//     // Handle any actions with this.set('myAction', function(val) { ... });
+
+//     await render(hbs`<Point />`);
+
+//     assert.equal(this.element.textContent.trim(), '');
+
+//     // Template block usage:
+//     await render(hbs`
+//       <Point>
+//         template block text
+//       </Point>
+//     `);
+
+//     assert.equal(this.element.textContent.trim(), 'template block text');
+//   });
+// });
```

Comment:
Any reason to comment?

Reply:
we don't have js file for point component, this test is added by ember cli, i should remove this.